.travis.yml

```diff
@@ -1,10 +1,3 @@
-branches:
```

Comment:
I think deleting this contradicts they way the bors workflow works. If we decide to not use bors, please remove also bors.toml.

Reply:
Why is it a contradiction? Now PRs are tested automatically and we can still use bors